src/TracingDriverManager.php

```diff
@@ -22,7 +22,6 @@ class TracingDriverManager extends Manager
     public function __construct($app)
     {
         parent::__construct($app);
-        $this->config = $this->app->make('config');
```

Comment:
@iyhunko Would this break compatibility with earlier versions of Laravel? If so please leave a comment in the PR, that the upcoming release would only support Laravel v8 so that is added in the release's description.

Reply:
@KinaneD yes, you are right. It will break compatibility with earlier versions of Laravel.